soc/st/stm32/common/stm32_wkup_pins.c

```diff
@@ -133,7 +131,31 @@ struct wkup_pin_cfg_t {
  * @brief LookUp Table to store LL_PWR_WAKEUP_PINx for each wake-up pin.
  */
 static const uint32_t table_wakeup_pins[PWR_STM32_MAX_NB_WKUP_PINS + 1] = {
-	LISTIFY(PWR_STM32_MAX_NB_WKUP_PINS, PWR_STM32_WKUP_PIN_DEF, (,))};
```

Comment:
I think LISTIFY could still be used by fixing it like this:
```
#define PWR_STM32_WKUP_PIN_DEF(i, _) _CONCAT1(LL_PWR_WAKEUP_PIN, UTIL_INC(i))
static const uint32_t table_wakeup_pins[PWR_STM32_MAX_NB_WKUP_PINS + 1] = {
	0,
	LISTIFY(PWR_STM32_MAX_NB_WKUP_PINS, PWR_STM32_WKUP_PIN_DEF, (,))};
```
I've done something similar here https://github.com/zephyrproject-rtos/zephyr/pull/82955/commits/c29a8b71b37e1db6d605b546cd3303703d2d5b9d